src/cluster_asm.c

```diff
@@ -3328,10 +3372,14 @@ void asmActiveTrimCycle(void) {
         jemalloc_purge();
 #endif
         asmActiveTrimEnd();
-    }
 
-    long long elapsed = ustime() - start;
-    latencyAddSampleIfNeeded("trim-cycle-slow", elapsed / 1000);
```

Comment:
I guess we don't need it for now, perhaps till we realize that we need it. (I feel like no need to make this API change for now)